### ISSUE TITLE
Smoke test Caseflow judge assignment to attorneys with VACOLS UI 

### ISSUE THREAD

**Issue by amprokop** [department-of-veterans-affairs/caseflow#5932]
## Context

We should do a manual test with Caseflow and the VACOLS UI before launching the judge assignment to attorney flow to ensure basic correctness and backwards compatibility.

This test does not have to test every detail and edge case, it should just act as a smoke test.

We should use Caseflow preprod, since Caseflow preprod connects to VACOLS dev and the VACOLS UI is running connected to the VACOLS DB. 

## Testing Steps
1. Log into Caseflow in preprod as BVAJWAGNER
1. Log into the VACOLS UI as BVAJWAGNER
1. Go through the VACOLS flow to assign a case to an attorney
1. Go through the Caseflow flow to assign a different case to the same attorney
1. Log in as the attorney in VACOLS - are both cases visible - does everything look the same?
1. Log in as the attorney in Caseflow - are both cases visible - does everything look the same?

**Comment by evankroske** [comment]
Judge assignment to attorneys has failed testing twice.

The first time, when I tried to assign tasks to attorneys, Caseflow showed an error message. The problem was caused by invalid data in the staff record of CASEFLOW1/DSUSER.

The second time, the assignment succeeded according to the API, but the changes weren't reflected in VACOLS. Refreshing the page undid any assignments. In this case, the problem was that one of the fake users had an invalid staff record.

The next test is scheduled for Friday morning.
